nrf_modem/doc/gnss_interface.rst

```diff
@@ -0,0 +1,529 @@
+
+By default, all QZSS PRNs (193...202) are enabled.
+
+QZSS PRNs 193, 194, 195 and 199 can be enabled (and others disabled) as shown in the following example:
+
+.. code-block:: c
+
+   err = nrf_modem_gnss_qzss_prn_mask_set(0x47);
+
+.. note::
+
+   QZSS is only supported by modem firmware v1.3.0 or later.
+
+Enabling GNSS priority mode
+***************************
+
+GNSS can be given priority over LTE idle mode procedures to help getting a fix.
+Usually, this is not necessary when either eDRX or PSM (or both) is used, but if that is not possible, the GNSS priority mode may be used.
+
+Priority for GNSS should be used only when a fix has been blocked by LTE idle mode operations, which can be detected by :c:data:`NRF_MODEM_GNSS_PVT_FLAG_NOT_ENOUGH_WINDOW_TIME` bit being set in the PVT data frame flags member.
+The application should not make the decision based on a single PVT event, but should enable priority only in case this flag has been set in several consecutive PVT events.
+
+Priority mode is disabled automatically after producing the first fix.
+It can also be disabled by the application by calling the :c:func:`nrf_modem_gnss_prio_mode_disable` function.
+
+.. note::
+
+   GNSS priority may interfere with LTE operations.
+   If possible, it would be good to time the use of priority to moments where data transfer is not anticipated.
+   In general, eDRX cycles that are long enough, or PSM, ensure better functionality for both GNSS and LTE.
+
+.. code-block:: c
+
+   err = nrf_modem_gnss_prio_mode_enable();
+
+Changing the dynamics mode
+**************************
+
+Dynamics mode can be used to tune GNSS performance for a specific use case.
+Using a matching dynamics mode improves the positioning performance.
+The dynamics mode can be changed without disruption in positioning.
+The selected dynamics mode is stored into the non-volatile memory.
+
+The default value is :c:data:`NRF_MODEM_GNSS_DYNAMICS_GENERAL_PURPOSE`.
+
+.. code-block:: c
+
+   err = nrf_modem_gnss_dyn_mode_change(NRF_MODEM_GNSS_DYNAMICS_AUTOMOTIVE);
+
+.. note::
+
+   Dynamics mode is only supported by modem firmware v1.3.0 or later.
+
+1PPS
+****
+
+GNSS can provide time synchronized eletrical pulses to the COEX1 pin.
+The rising edge of the pulse is aligned as closely as possible to the GPS time second.
+
+The pulse interval and width are configurable.
+It is also possible to configure the pulses to start at a specific date and time.
+Instead of repeating pulses, 1PPS can also be used in a one-time pulse mode, where only a single pulse is given at the specified time or as soon as GNSS gets a fix.
+
+GNSS only starts giving pulses after it has got at least one fix.
+After this, the pulses will continue also when GNSS is no longer running, but the precision will start degrading.
+
+In cases where GNSS is not running continuously it may be beneficial to change the timing source used by GNSS during sleep periods, see :ref:`sleep_timing_source`.
```

Comment:
```suggestion
In cases where GNSS is not running continuously, it may be beneficial to change the timing source used by GNSS during sleep periods, see :ref:`sleep_timing_source`.
```

Reply:
Ok.